manifests/l/leinelissen/aeon/0.2.18/leinelissen.aeon.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: leinelissen.aeon
+PackageVersion: 0.2.18
+PackageLocale: en-US
+Publisher: Lei Nelissen
+PublisherUrl: https://github.com/leinelissen
+PublisherSupportUrl: https://github.com/leinelissen/aeon/issues
+Author: leinelissen
+PackageName: Online identity versioning for the masses

```

```suggestion
PackageName: Aeon
```
This must match Control Panel.